frontend/src/components/room/chat/Chat.tsx

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useRef, useState } from "react";
```

Comment:
Can we call this a TextChat instead of Chat? To distinguish it from VideoChat

Reply:
It will also be good to separate the room folder files into videochat and textchat folders for housekeeping